extensions/default/src/DicomWebDataSource/utils/StaticWadoClient.ts

```diff
@@ -156,19 +164,28 @@ export default class StaticWadoClient extends api.DICOMwebClient {
    *
    * @param {*} desired
    * @param {*} actual
+   * @param {boolean} caseSensitive
    * @returns true if the values match
    */
-  compareValues(desired, actual) {
+  compareValues(desired, actual, caseSensitive = true) {
     if (Array.isArray(desired)) {
-      return desired.find(item => this.compareValues(item, actual));
+      return desired.find(item => this.compareValues(item, actual, caseSensitive));
     }
     if (Array.isArray(actual)) {
-      return actual.find(actualItem => this.compareValues(desired, actualItem));
+      return actual.find(actualItem => this.compareValues(desired, actualItem, caseSensitive));
     }
     if (actual?.Alphabetic) {
       actual = actual.Alphabetic;
     }
     if (typeof actual == 'string') {
+      if (!caseSensitive) {
+        if (typeof desired === 'string') {
+          desired = desired.toLowerCase();
+        }
+        if (typeof actual === 'string') {
```

Comment:
actual.toLowerCase is sufficient, and resolves one bug where === 'string' doesn't work correctly.